### PR TITLE
About Chinese localization (PR)

### LINKED ISSUE #3: About Chinese localization
In this [commit](https://github.com/Frederisk/Wikitext-Extension-Gadget/commit/064ba3d4044687bb6cae12499fa011aa3aec9d23), i18n was finally here but when it comes to Chinese (`zh`) the thing is not that right.
For Chinese (`zh`), the fallback is firstly to `zh-hans`, then `zh-hant`. But actually the reason to have a language code `zh` is that in this language, Language Converter is given to automatically convert the page content from Simp. to Trad. or vice versa.
My testing result is that when users choose `zh`, it actually uses Simplified Chinese, thus in [this line](https://github.com/Frederisk/Wikitext-Extension-Gadget/commit/064ba3d4044687bb6cae12499fa011aa3aec9d23#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R55) it should actually be `'zh': i18nSource['simplified_chinese'],`.

----
It's just a code fact that the `zh` language class in MediaWiki actually inherits from `zh-hans`. I do not welcome any political comment on this and sorry if you feel a bit not comfortable. I'm learning language differences as well and from my knowledge `zh-hant` uses `開啟` for open. It seems like you are native in `zh-tw` so I wouldn't feel surprised if you say that it's not the case.

### PR DESCRIPTION
**Pull request by lakejason0**
See #3 .

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -36,8 +36,8 @@ jQuery.when(window.mw.loader.using('mediawiki.util'), $.ready).then((): void =>
       tooltip: '在 Visual Studio Code 中打开此页',
     },
     traditional_chinese: {
-      text: '在 VSCode 中打開',
-      tooltip: '在 Visual Studio Code 中打開此頁',
+      text: '在 VSCode 中開啟',
+      tooltip: '在 Visual Studio Code 中開啟此頁',
```
Comment (Frederisk) [inline src/index.ts:40]: Both "開啟" and "打開" are feasible.

File: src/index.ts [hunk 2]
```diff
@@ -52,7 +52,7 @@ jQuery.when(window.mw.loader.using('mediawiki.util'), $.ready).then((): void =>
     'ko': i18nSource['korean'],
     'yue': i18nSource['cantonese'],
     'zh-yue': i18nSource['cantonese'],
-    'zh': i18nSource['traditional_chinese'],
+    'zh': i18nSource['simplified_chinese'],
```
Comment (Frederisk) [inline src/index.ts:55]: I don't understand, can you give the source code of the corresponding practice? But keeping Traditional Chinese is just my personal preference.

**Comment by Frederisk** [comment]
These codes are basically produced by GitHub Copilot, and many places need to be improved, but I still need free time to do it.

**Review by Frederisk — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (lakejason0) [reply src/index.ts:40]: ok.

File: src/index.ts (hunk 2 above)
Comment (lakejason0) [reply src/index.ts:55]: (Inheritance Diagram)[https://doc.wikimedia.org/mediawiki-core/master/php/classLanguageZh.html] from WMF docs.

Comment (lakejason0) [reply src/index.ts:55]: I mean, you may again request the variant the user is using but from my experience most of the gadgets just read from the interface language.

Comment (Frederisk) [reply src/index.ts:55]: Yes, this **shall** follow the user interface language.

**Review by Frederisk — APPROVED** [review]
And do you have anything else?  If not I will merge the fork.

**Comment by lakejason0** [comment]
> And do you have anything else? If not I will merge the fork.

Not really. Thank you for your work!

**Comment by Frederisk** [comment]
> Not really.

What? Anything else? Or you mean "really bot"?

**Comment by lakejason0** [comment]
> > Not really.
> 
> What? Anything else? Or you mean "really not"?

Nothing else, sorry.

**Pull request merged**